cpp/src/io/parquet/page_data.cu

```diff
@@ -815,6 +815,39 @@ inline __device__ void gpuOutputDecimalAsFloat(volatile page_state_s *s,
   *dst  = (scale < 0) ? (d * kPow10[min(-scale, 39)]) : (d / kPow10[min(scale, 39)]);
 }
 
+/**
+ * @brief Output a fixed-length byte array(len <= 8) as a 64-bit int
+ *
+ * @param[in,out] s Page state input/output
+ * @param[in] src_pos Source position
+ * @param[in] dst Pointer to row output data
+ */
+inline __device__ void gpuOutputFixedLenByteArrayAsInt64(volatile page_state_s *s,
+                                                         int src_pos,
+                                                         int64_t *dst)
+{
+  uint32_t const dtype_len_in = s->dtype_len_in;
+  uint8_t const *dict         = s->dict_base ? s->dict_base : s->data_start;
+  uint32_t const dict_pos =
```

Comment:
A bit questionable naming of variables here. It appears that depending on the encoding, the `dict` ptr points to either the dict or the data. Similarly for the variable `dict_pos`. Could've been `data` and `pos`. But this seems to be the trend in the entire file.

Reply:
Yes, @devavret. Comparing to `dict` and `dict_pos`,  `data` and `pos` are better names.  The name style here is just following the taste of other decoding methods. Do you think we need to change to  `data` and `pos` ?

Reply:
This is just something we'll need to remember during the refactor.

Reply:
Yes please. Just change the ones in this function. Some functions are doing this correctly https://github.com/rapidsai/cudf/blob/f117b68ca68faa0acd8f9fa485cad0aa63a0e543/cpp%2Fsrc%2Fio%2Fparquet%2Fpage_data.cu#L627-L629